samples/04_gis_analysts_data_scientists/calculate_impervious_surfaces_from_spectral_imagery.ipynb

```diff
@@ -391,7 +391,7 @@
     }
```

Comment:
<p>Error:</p><pre class="ql-syntax" spellcheck="false">Unable to apply the current raster function template on the imagery layer. This might be because the server could not process the template, the template is invalid or not populated with correct arguments. (Make sure that Raster rendering service is turned on, inorder to display the output dynamically.)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1019/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>